client/signup/steps/select-plan/index.jsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import PlansStepComponent from 'signup/steps/plans';
+
+module.exports = React.createClass( {
+	displayName: 'SelectPlanStep',
```

Comment:
Pretty minor, but what do you think about giving this a more descriptive name? It isn't clear why `PlansStep` and `SelectPlansStep` would be different. Maybe some variant of `PaidPlansStep`?